src/controller/SetUpCodePairer.cpp

```diff
@@ -244,37 +245,26 @@ CHIP_ERROR SetUpCodePairer::StartDiscoveryOverDNSSD()
 
 CHIP_ERROR SetUpCodePairer::StopDiscoveryOverDNSSD()
 {
-    ChipLogDetail(Controller, "Stopping commissioning discovery over DNS-SD");
+    ChipLogProgress(Controller, "Stopping commissionable node discovery over DNS-SD");
 
     mWaitingForDiscovery[kIPTransport] = false;
 
     mCommissioner->StopCommissionableDiscovery();
     return CHIP_NO_ERROR;
 }
 
-CHIP_ERROR SetUpCodePairer::StartDiscoveryOverSoftAP()
-{
-    return CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE;
-}
-
-CHIP_ERROR SetUpCodePairer::StopDiscoveryOverSoftAP()
-{
-    mWaitingForDiscovery[kSoftAPTransport] = false;
-    return CHIP_NO_ERROR;
-}
-
```

Comment:
Completely dead code in private impl details, for something removed from the main spec.